static/app/components/forms/controls/selectOption.tsx

```diff
@@ -56,17 +56,28 @@ function SelectOption(props: Props) {
           innerWrapProps={{'data-test-id': value}}
           labelProps={{as: typeof label === 'string' ? 'p' : 'div'}}
           leadingItems={
-            <Fragment>
-              <CheckWrap isMultiple={isMultiple} isSelected={isSelected}>
-                {isSelected && (
-                  <IconCheckmark
-                    size={isMultiple ? 'xs' : 'sm'}
-                    color={isMultiple ? 'white' : undefined}
-                  />
-                )}
-              </CheckWrap>
-              {data.leadingItems}
-            </Fragment>
+            itemProps.__isNew__ ? (
+              <Fragment>
+                <IconAdd size="sm" />
+                {data.leadingItems}
+              </Fragment>
+            ) : isMultiple ? (
+              <Fragment>
+                <CheckWrap isMultiple isSelected={isSelected}>
+                  {isSelected && <IconCheckmark size="xs" color="white" />}
+                </CheckWrap>
+                {data.leadingItems}
+              </Fragment>
+            ) : (
+              isSelected && (
+                <Fragment>
+                  <CheckWrap isMultiple={false} isSelected>
+                    <IconCheckmark size="sm" />
+                  </CheckWrap>
+                  {data.leadingItems}
+                </Fragment>
```

Comment:
I've implemented this change because the values looked odd when no item was selected—they were shifted to the right. @vuluongj20, could you please review and let me know if this works for you?

**Before:**
![image](https://github.com/user-attachments/assets/e2f52c0c-1cf1-4ee3-a95d-ead86987e162)


**After:**

![image](https://github.com/user-attachments/assets/3ea58510-c49f-4af4-8609-3e6ac9dd2bf6)
![image](https://github.com/user-attachments/assets/5b3b4583-0559-4ba3-bf3c-96679f9c6480)

Reply:
^ @shellmayr

Reply:
Looks much better now 🙏

Reply:
I removed this change from this PR and created a [new one](https://github.com/getsentry/sentry/pull/75434) 😉